hack/configure-self-signed-cluster.md

```diff
@@ -7,17 +7,11 @@ With local development clusters, often the cluster certificate is self-signed ra
 From OpenShift, the default ingress cert is in `-n openshift-ingress`
 
 ```bash
-mkdir clustercert && cd clustercert
+sudo mkdir /var/run/clustercert && cd /var/run/clustercert
```

Comment:
this can stay non-root & in non-root location (as is like current dir, or, /tmp/clustercert)